sdk/helper/scaleutils/nodeselector/oldest_create_index_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package nodeselector
+
+import (
+	"testing"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_oldestClusterScaleInNodeSelectorName(t *testing.T) {
+	assert.Equal(t, "oldest_create_index", newOldestCreateIndexClusterScaleInNodeSelector().Name())
```

Comment:
We're trying to gradually migrate Nomad repos to use `shoenig/test` over testify:

```suggestion
	must.Eq(t, "oldest_create_index", newOldestCreateIndexClusterScaleInNodeSelector().Name())
```